apps/generator/test/test-project/test.sh

```
@@ -51,7 +51,7 @@ Starting registry test
 sudo chown -R 1001:121 "/root/.npm"
 
 # Always run these steps
-cd app
+cd /app
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)